src/core/storage/utils.ts

```diff
@@ -1,20 +1,15 @@
-import { CarReader } from '@ipld/car';
-import { importDAG } from '@ucanto/core/delegation';
 import { Capabilities, Delegation } from '@ucanto/interface';
 import { lookup } from 'mime-types';
+import * as Proof from '@web3-storage/w3up-client/proof';
 
 /**
  * Parses a delegation from a base64 encoded CAR file
  * @param data - The base64 encoded CAR file
  * @returns The parsed delegation
  */
 export async function parseDelegation(data: string): Promise<Delegation<Capabilities>> {
-  const blocks = [];
-  const reader = await CarReader.fromBytes(Buffer.from(data, 'base64'));
-  for await (const block of reader.blocks()) {
-    blocks.push(block);
-  }
-  return importDAG(blocks);
+  const proof = await Proof.parse(data);
+  return proof;
 }
```

Comment:
Personally I'd just get rid of this entire function and just inline `Proof.parse`.

Reply:
That makes sense. I will keep it as is for now just because I don't want tweak all the vi mocks in the tests at this moment. Thanks for the suggestion.